src/main/java/org/sagebionetworks/bridge/spring/controllers/ParticipantDataController.java

```diff
@@ -0,0 +1,63 @@
+package org.sagebionetworks.bridge.spring.controllers;
+
+import org.sagebionetworks.bridge.models.ForwardCursorPagedResourceList;
+import org.sagebionetworks.bridge.models.ParticipantData;
+import org.sagebionetworks.bridge.models.StatusMessage;
+import org.sagebionetworks.bridge.models.accounts.UserSession;
+import org.sagebionetworks.bridge.services.ParticipantDataService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.sagebionetworks.bridge.BridgeConstants.API_DEFAULT_PAGE_SIZE;
+import static org.sagebionetworks.bridge.BridgeUtils.getIntOrDefault;
+
+@CrossOrigin
+@RestController
+public class ParticipantDataController extends BaseController {
+
+    private ParticipantDataService participantDataService;
+
+    @Autowired
+    final void setReportService(ParticipantDataService participantDataService) {
+        this.participantDataService = participantDataService;
+    }
+
+    @GetMapping("/v3/users/self/configs")
```

Comment:
Since we're calling this ParticipantData, we sould change the url from /configs to /data (here and in the rest of the URLs).

Reply:
Oops duh! Thanks, I’ll change it.